extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -117,6 +125,51 @@ export class Bind {
      */
     this.digestQueuedAfterScan_ = false;
 
+    /**
+     * @const @private {!Array<Node>}
+     */
+    this.dynamicRoots_ = [];
+
+    /**
+     * @const @private {!Array<Promise>}
+     */
+    this.mutationPromises_ = [];
+
+    this.subtreeMutationObserver_ = new MutationObserver(mutations => {
```

Comment:
I think it's ok to assume availability here. It has the same browser support as [web workers](http://caniuse.com/#feat=webworkers).